crane/views/v2.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import absolute_import
+from flask import Blueprint, json, current_app, redirect
+
+from crane import app_util
+from crane.api import repository
+
+
+section = Blueprint('v2', __name__, url_prefix='/v2')
+
+
+@section.after_request
+def add_common_headers(response):
+    """
+    Add headers to a response.
+
+    All 200 responses get a content type of 'application/json', and all others
+    retain their default.
+
+    Headers are added to make this app look like the actual docker-registry.
+
+    :param response:    flask response object for a request
+    :type  response:    flask.Response
+
+    :return:    a response object that has the correct headers
+    :rtype:     flask.Response
+    """
+    # if response code is 200, assume it is JSON
+    if response.status_code == 200:
+        response.headers['Content-Type'] = 'application/json'
+        response.headers['Docker-Distribution-API-Version'] = 'registry/2.0'
+    return response
+
+
+@section.route('/')
+def v2():
+    """
+    Provides version support information for /v2 requests.
+
+    :return: Empty JSON document
```

Comment:
Are you sure this doesn't return a flask response object? Either way, this needs an "rtype" for clarity.